src/index.js

```diff
@@ -28,6 +42,7 @@ function zulip(initialConfig) {
     return parseConfigFile(initialConfig.zuliprc).then(config => resources(config));
   }
   const config = initialConfig;
+  globalconfig = config;
```

Comment:
I am not the brightest at JavaScript, but what's the purpose of `globalconfig` here?

Reply:
I didn't go into using `this.config` and `config` because of readability. The `callEndpoint` function added above uses the config and it is not being passed to it, so I needed to store the config in a global variable once generated.